src/js/records/requests/asset-update.record.js

```diff
@@ -1,11 +1,18 @@
 import { ASSET_POLICIES } from '@tokend/js-sdk/lib/index'
+
 import { RequestRecord } from '../request-record'
+import { AssetRecord } from '../entities/asset.record'
+
 import _get from 'lodash/get'
 
 export class AssetUpdateRequestRecord extends RequestRecord {
   constructor (record, details) {
     super(record)
 
+    this.asset = new AssetRecord(_get(
+      this._record, 'details.assetUpdate'
```

Comment:
Same here, I'm not voting for embedding the `asset` resource inside `asset request` resource.

Reply:
https://github.com/tokend/web-client/pull/37#discussion_r254729398